test/orders/service/JsonOrderFormatterService.spec.ts

```diff
@@ -56,6 +58,23 @@ describe('JsonOrderFormatterService', () => {
         const order = JsonOrderFormatterService.formatOrder(singleOrder.default);
         expect(order.id).to.equal('c0f45529-cbed-4e90-9a38-c208d409ef2a');
         expect(order.updatedAt.getFullYear()).to.equal(2018);
+        expect (order.lineItems.length).to.equals(2);
+        expect (order.lineItems[0].taxes).to.not.be.undefined;
+        // @ts-ignore
```

Comment:
Sorry about this syntax but i couldn't find a better way to do it. I didn't want to put nested if checks in the tests since I think that looks a little more obscure.